### PR TITLE
AG-12145 Chart type style updates

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-12145

### CODE REVIEW

**Review by alantreadway — APPROVED** [review]
LGTM - feedback from design required.

File: packages/ag-charts-community/src/chart/dom/theme.css [hunk 1]
```diff
@@ -64,10 +64,16 @@
...
     /* Popover */
-    --ag-charts-popover-background-color: #fafafb;
-    --ag-charts-popover-border: 1px solid var(--ag-charts-border-color);
-    --ag-charts-popover-active-color: var(--ag-active-color, #5090dc);
-    --ag-charts-popover-active-background-color: #e1f0fb;
+    --ag-charts-popover-background-color: var(--ag-charts-toolbar-background-color);
+    --ag-charts-popover-foreground-color: var(--ag-charts-toolbar-foreground-color);
+    --ag-charts-popover-border: var(--ag-charts-toolbar-border);
+    --ag-charts-popover-focus-color: var(--ag-charts-toolbar-focus-color);
+    --ag-charts-toolbar-hover-color: color-mix(
```
Comment (markdurrant) [inline packages/ag-charts-community/src/chart/dom/theme.css]: Very minor, but we should move this to the 'toolbar' section of the file.

**Review by markdurrant — CHANGES_REQUESTED** [review]
Generally LGTM, one very minor change.

File: packages/ag-charts-community/src/chart/dom/theme.css (hunk 1 above)
Comment (jacobp100) [reply packages/ag-charts-community/src/chart/dom/theme.css]: Ah yes my bad. Updated

Comment (markdurrant) [reply packages/ag-charts-community/src/chart/dom/theme.css]: 👍

**Review by markdurrant — APPROVED** [review]
All LGTM now 👍

**Pull request merged**